package.nls.json

```diff
@@ -10,7 +10,7 @@
     "containerApps.browse": "Browse",
     "containerApps.openInPortal": "Open in Portal",
     "containerApps.createContainerApp": "Create Container App...",
-    "containerApps.deployImage": "Deploy Image to Container App...",
+    "containerApps.deployImage": "Update Image in Container App...",
```

Comment:
What was the reasoning for this verbiage change?

Reply:
The command kind of changed.  It used to add a new image to the container app.  Each container app could have 4 images running simultaneously, so the verb deployed made more sense.

However, we decided that's a pretty edge use case, so we are now removing the old image and replacing it with the new image, so it functions more like an update.

Reply:
Would you be okay with holding off a bit? @anthonychu and I are still kind of trying to figure out the best way to convey what this command does.  I was planning to rename everything (the file name, localize labels, etc.) but I didn't want to do that multiple times so was just waiting until we landed on something

Reply:
Sure thing. Definitely not a blocker for this 👍